src/couch/src/couch_hash.erl

```diff
@@ -31,15 +31,27 @@ md5_hash_update(Context, Data) ->
 -else.
 
 md5_hash(Data) ->
-    crypto:hash(md5, Data).
+    case config:is_enabled(fips_mode) of
```

Comment:
For fips mode feature toggle. We'd have to make sure we set it as early as possible. It turns out some couch_epi code uses hashes early. So we may have to move that to config's init